src/WPF/ArcGISRuntime.WPF.Viewer/Samples/LocalServer/LocalServerGenerateElevationProfile/LocalServerGenerateElevationProfile.xaml.cs

```diff
@@ -0,0 +1,598 @@
+
+                // Instantiate the elevation profile feature layer from the feature table.
+                _elevationProfileFeatureLayer = new FeatureLayer(featureTable);
+
+                // Load the elevation profile feature layer.
+                await _elevationProfileFeatureLayer.LoadAsync();
+
+                // Set the surface placement and renderer for the elevation profile feature layer.
+                _elevationProfileFeatureLayer.SceneProperties.SurfacePlacement = SurfacePlacement.Absolute;
+                _elevationProfileFeatureLayer.Renderer = new SimpleRenderer(new SimpleLineSymbol(SimpleLineSymbolStyle.Solid, System.Drawing.Color.White, 3));
+
+                Dispatcher.Invoke(() =>
+                {
+                    // Add the elevation profile feature layer to the scene.
+                    MySceneView.Scene.OperationalLayers.Add(_elevationProfileFeatureLayer);
+
+                    // Update the UI.
+                    MyInstructionsContentTextBlock.Text = "Elevation Profile drawn.";
+                    MyGenerateElevationProfileButton.Visibility = Visibility.Visible;
+                    MyDrawPolylineButton.Visibility = Visibility.Visible;
+                    MyClearResultsButton.Visibility = Visibility.Visible;
+                    MyClearResultsButton.IsEnabled = true;
+                    MyCancelJobButton.Visibility = Visibility.Collapsed;
+                    MyProgressBar.Visibility = Visibility.Collapsed;
+                    MyProgressBarLabel.Visibility = Visibility.Collapsed;
+                });
+
+                // Set the viewpoint based on the position of the elevation profile.
+                await MySceneView.SetViewpointCameraAsync(CreateCameraFacingElevationProfile(), new TimeSpan(0, 0, 0, 2, 0));
+            }
+            catch (Exception e)
+            {
+                MessageBox.Show(e.Message, "Error");
+            }
+            finally
+            {
+                // Remove the event handlers.
+                _gpJob.ProgressChanged -= GpJob_ProgressChanged;
+                _gpJob.StatusChanged -= GpJob_StatusChanged;
+            }
+        }
+
+        private Camera CreateCameraFacingElevationProfile()
+        {
+            // Get the polyline's end point coordinates.
+            MapPoint endPoint = _polyline.Parts[0].EndPoint;
+            double endPointX = endPoint.X;
+            double endPointY = endPoint.Y;
+
+            // Get the polyline's center point coordinates.
+            MapPoint centerPoint = _polyline.Extent.GetCenter();
+            double centerX = centerPoint.X;
+            double centerY = centerPoint.Y;
+
+            // Calculate the position of a point perpendicular to the centre of the polyline.
+            double lengthX = endPointX - centerX;
+            double lengthY = endPointY - centerY;
+            MapPoint cameraPositionPoint = new MapPoint(centerX + lengthY * 2, centerY - lengthX * 2, 1200,
```

Comment:
```suggestion
            var cameraPositionPoint = new MapPoint(centerX + lengthY * 2, centerY - lengthX * 2, 1200,
```
IN our style we can use `var` when it precedes a constructor.